RubberduckTests/Inspections/ImplicitActiveSheetReferenceInspectionTests.cs

```diff
@@ -106,21 +125,33 @@ public void ImplicitActiveSheetReference_ReportsInWorkbookModules()
     arr1 = Cells(1,2)
 End Sub
 ";
-            var module = ("Sheet1", inputCode, ComponentType.Document);
-            var vbe = MockVbeBuilder.BuildFromModules(module, ReferenceLibrary.Excel).Object;
-
-            using (var state = MockParser.CreateAndParse(vbe))
+            var modules = new (string, string, ComponentType)[]
             {
-                var documentModule = state.DeclarationFinder.UserDeclarations(DeclarationType.Document)
-                    .OfType<DocumentModuleDeclaration>()
-                    .Single();
-                documentModule.AddSupertypeName("Workbook");
-
-                var inspection = InspectionUnderTest(state);
-                var inspectionResults = inspection.GetInspectionResults(CancellationToken.None);
+                ("ThisWorkbook", inputCode, ComponentType.Document),
+                ("Sheet1", string.Empty, ComponentType.Document),
+                ("Class1", string.Empty, ComponentType.ClassModule)
+            };
+            Assert.AreEqual(1, InspectionResultsForModules(modules, ReferenceLibrary.Excel, DefaultDocumentModuleSupertypeNames).Count());
+        }
 
-                Assert.AreEqual(1, inspectionResults.Count());
-            }
+        [Test]
+        [Category("Inspections")]
+        public void ImplicitActiveSheetReference_NoResultForWorksheetVariable()
+        {
```

Comment:
Can we please have a corresponding test for `ImplicitContainingSheetReferenceInspection` as well?

Reply:
Good call, will add.